contributing.md

```diff
@@ -1,9 +1,126 @@
-# Contributing to the p5.js web editor
+# Contributing to the p5.js web editor 
 
-We welcome community contributions to the p5.js web editor! You can contribute even if you are not a developer by documenting bugs, proposing new features, and updating documentation.
+Hi! Thanks for wanting to contribute to P5's web editor! You are awesome! 😊
```

Comment:
I want to keep the emphasis that contributing is not just writing code. Could you change this to "Hello! We welcome community contributions to the p5.js web editor. Contributing takes many forms and doesn't have to be writing code, it can be documenting bugs, proposing new features, and updating documentation."